aeron-agent/src/test/java/io/aeron/agent/ArchiveEventLoggerTest.java

```diff
@@ -66,7 +66,8 @@ void after()
         names = {
             "CMD_OUT_RESPONSE", "REPLICATION_SESSION_STATE_CHANGE",
             "CONTROL_SESSION_STATE_CHANGE", "REPLAY_SESSION_ERROR", "CATALOG_RESIZE",
-            "REPLICATION_SESSION_DONE"
+            "REPLICATION_SESSION_DONE", "REPLAY_SESSION_STATE_CHANGE", "RECORDING_SESSION_STATE_CHANGE",
+            "REPLICATION_SESSION_STARTED", "REPLAY_SESSION_STARTED", "RECORDING_SESSION_STARTED"
```

Comment:
Not needed. Covered by `CMD_IN_REPLICATE`, `CMD_IN_REPLICATE2` and `CMD_IN_TAGGED_REPLICATE`.
Please check those events that they log enough details.